app/frontend/Calendar/EditEvent/EditEvent.svelte

```diff
@@ -93,7 +93,7 @@
 
   export let event: Event;
 
-  $: showRepeat = $event.recurrenceCase != RecurrenceCase.Normal;
```

Comment:
Why is this condition no longer correct?
(Could it be modified slightly to remain here?)

Reply:
> I think I could restore it without the $ by making some other changes.

With the code comment that I added, it's OK for now.

Change it if (and only if) your change makes the code easier to understand or working more reliably in case of future code changes.

Reply:
This does simplify matters, thanks; I should be able to implement that to fix issues #591 and #592. (This will also result in a further change to this condition.)

Reply:
> This does simplify matters, thanks

Great!

> (This will also result in a further change to this condition.)

That's OK, as long as the changes are inherent. I just want them to make sense independent from the UI.